### PR TITLE
Milestone1

### PR DESCRIPTION
**Pull request by Greg0109**
Changelog:
- Added function to add books
- Added function to remove books
- Books will remain in localstorage

### CODE REVIEW

**Review by Temmarie — COMMENTED** [review]
## Status: Invalid ❗ 

### Hi @hiromataba and @Greg0109   👋🏾 , 

 **While you made a great effort in this project, unfortunately, I cannot proceed to review your code.**

### Invalid Code Review Request

I have to mark this submission as invalid because you skipped the following: 
- [x] You have to use the correct GitHub flow for this project. You will need to create a pull request to compare your `feature` branch with the `main` branch. Kindly make sure to use a more descriptive name for your branches. See [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) for more information.
- [x] Kindly add a descriptive readme to your project.





#### Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

#### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            <br>
+            <label for="author">Author</label>
+            <br>
+            <input type="text" name="author" id="author">
+        </form>
+        <p class="btn-danger w-100 text-center mt-5" id="add-button">Add Book</p>
+    </div>
+
+    
+    <script src="/JS/main.js"></script>
```
Comment (vichuge) [inline index.html]: - It looks like you got errors with your URL. I kindly invite you to fix it. Hint: Use GitHub pages to see the bug in an easier way. 🐶 
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE-edge">
+    <meta name="viewpoint" content="width-device-width, initial-scale=1.0">
+    <title>Books</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="/CSS/main.css">
```
Comment (vichuge) [inline index.html]: - It looks like you got errors with your URL. I kindly invite you to fix it. Hint: Use GitHub pages to see the bug in an easier way. 🐶 
image

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shaher-11 — APPROVED** [review]
#### Hello @Greg0109,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**